src/test/groovy/com/stormpath/tck/login/SocialLoginIT.groovy

```diff
@@ -0,0 +1,146 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stormpath.tck.login
+
+import com.jayway.restassured.http.ContentType
+import com.jayway.restassured.path.xml.XmlPath
+import com.jayway.restassured.path.xml.element.Node
+import com.jayway.restassured.path.xml.element.NodeChildren
+import com.jayway.restassured.response.Response
+import com.stormpath.tck.AbstractIT
+import com.stormpath.tck.util.Iso8601Utils
+import com.stormpath.tck.util.*
+import org.testng.annotations.BeforeClass
+import org.testng.annotations.Test
+
+import static com.jayway.restassured.RestAssured.delete
+import static com.jayway.restassured.RestAssured.given
+import static com.jayway.restassured.RestAssured.put
+import static org.hamcrest.Matchers.*
+import static org.testng.Assert.*
+
+@Test
+class SocialLoginIT extends AbstractIT {
+    // We are really only going to try testing LoginWithFacebook for now, so might have some things hardcoded.
+
+    private String loginRoute = "/login"
+
+    private String facebookClientID
+    private String facebookClientSecret
+
+    private String facebookTestUserAccessToken
+    private String facebookTestUserEmail
+
+    @BeforeClass
+    private void getSocialProviderTokens() throws Exception {
+        getSocialProviderIdsFromStormpath()
+        getFacebookAccessToken()
+    }
+
+    private void getSocialProviderIdsFromStormpath() {
+        // Pull account stores
+        List<String> accountStores = given()
+            .header("User-Agent", "stormpath-framework-tck")
+            .header("Authorization", RestUtils.getBasicAuthorizationHeaderValue())
+            .port(443)
+        .when()
+            .get(EnvUtils.stormpathApplicationHref + "/accountStoreMappings")
+        .then()
+            .statusCode(200)
+        .extract().path("items.accountStore.href")
+
+        // Find the FB account store, then put its client id in the class
+        accountStores.each { accountStore ->
+            Map<String> provider = given()
+                .header("User-Agent", "stormpath-framework-tck")
+                .header("Authorization", RestUtils.getBasicAuthorizationHeaderValue())
+                .port(443)
+            .when()
+                .get(accountStore + "?expand=provider")
+            .then()
+                .statusCode(200)
+            .extract().path("provider")
+
+            if (provider.get("providerId") == "facebook") {
+                this.facebookClientID = provider.get("clientId")
+                this.facebookClientSecret = provider.get("clientSecret")
+                return
+            }
+        }
+    }
+
+    private void getFacebookAccessToken() {
+        // We create a FB test user, and get its access token. We don't clean it up but oh well, TODO? 
```

Comment:
It's just an authenticated DELETE on the url, but I don't see a huge amount of value, as leaving the test user won't interfere with later runs of the test.